PowerEditor/src/NppIO.cpp

```diff
@@ -293,11 +293,10 @@ BufferID Notepad_plus::doOpen(const generic_string& fileName, bool isRecursive,
 
 		if (buffer != BUFFER_INVALID)
 		{
-			bool isSnapshotMode = (backupFileName != NULL and PathFileExists(backupFileName));
+			isSnapshotMode = (backupFileName != NULL && ::PathFileExists(backupFileName));
 			if (isSnapshotMode)
 			{
 				// To notify plugins that a snapshot dirty file is loaded on startup
-				SCNotification scnN;
 				scnN.nmhdr.hwndFrom = 0;

```

Comment:
Line #280 = scnN.nmhdr.hwndFrom = _pPublicInterface->getHSelf();
Line #301 = scnN.nmhdr.hwndFrom = 0;
Line #325 = Unspecified, changes from #280 to #301